### DISCUSSION TITLE
Measured Quantization probability differences for Mistral 7b and Llama 13b

### DISCUSSION THREAD

**Discussion by kalomaze** [ggml-org/llama.cpp#4183] [General]
https://www.reddit.com/r/LocalLLaMA/comments/1816h1x/how_much_does_quantization_actually_impact_models/

Mistral 7b average KL divergence:
![image](https://github.com/ggerganov/llama.cpp/assets/66376113/19f6fd8a-2b16-4ec0-9176-6c7079f515c3)

Llama 13b average KL divergence:
![image](https://github.com/ggerganov/llama.cpp/assets/66376113/898b5be0-cadb-414f-a153-2a82f8c103c2)

**Comment by BarfingLemurs** [comment]
Does this work as a benchmark between models trained on the same data, like llama 7B and 13B?

**Reply by kalomaze** [reply]
Depends on whether or not you consider the base model of 13b objectively superior in every way, which is hard to quantify I'd assume. At its core, the graph is only measuring how different each quantization is from the base model on average
